docs/mm_quick_start.md

```diff
@@ -0,0 +1,181 @@
+
+You will note that the top model configuration has a lower total combined throughput than either the 2nd or 3rd configurations. However, it does a better job at increasing the total percentage throughput gain versus the other configurations.
+
+For example, comparing the top config (42,24) to the third best (37,19), you will see that while the gain in `add_sub` throughput is large (~1600 infer/sec) it is only a 3.5% increase. However, this is offset by a decrease in `resnet50_libtorch` throughput (~75 infer/sec), which represents an 8% decrease in throughput. Therefore, the (42,24) is deemed to be a better configuration by MA.
+
+---
+
+The measured data and summary report will be placed inside the
+`./profile_results` directory. The directory will be structured as follows.
+
+```
+$HOME
+  |--- model_analyzer
+              |--- profile_results
+              .       |--- plots
+              .       |      |--- simple
+              .       |      |      |--- add_sub,resnet50_libtorch
+                      |      |              |--- gpu_mem_v_latency.png
+                      |      |              |--- throughput_v_latency.png
+                      |
+                      |--- results
+                      |       |--- metrics-model-inference.csv
+                      |       |--- metrics-model-gpu.csv
+                      |       |--- metrics-server-only.csv
+                      |
+                      |--- reports
+                              |--- summaries
+                              .        |--- add_sub,resnet50_libtorch
+                              .                |--- result_summary.pdf
+```
+
+## `Optional:` Add constraints
+
+---
+
+In the above results you can see a large (472%) throughput increase, but this is at the expense of a large increase in latency. By creating a YAML config file we can specify per model latency constraints:
+
+```
+profile_models:
+  add_sub:
+    constraints:
+      perf_latency_p99:
+        max: 3
+  resnet50_libtorch:
+    constraints:
+      perf_latency_p99:
+        max: 20
+```
+
+And then re-run with the following command:
+
+```
+model-analyzer profile \
+    --model-repository <path-to-examples-quick-start> \
+    --config-file <path-to-config-file> --triton-launch-mode=docker \
+    --output-model-repository-path <path-to-output-model-repo>/<output_dir>
+    --export-path profile_results
+    --run-config-profile-models-concurrently-enable
+```
+
+---
+
+Here is an example result summary with the above constraints, run on a TITAN RTX GPU:
+![Result Summary Constraint Top](../examples/mm_result_summary_constraint_top.jpg)
+![Result Summary Constraint Table](../examples/mm_result_summary_constraint_table.jpg)
+
+---
+
+You will observe that the top configuration here has a higher total combined throughput than the unconstrained run, however the total percentage gain is lower due to the large decrease in throuhgput for the `resnet50_libtorch` model.
```

Comment:
throughput

Reply:
is there a way to run the whole thing through spell check?

Reply:
Yes, I found a spell check extension and fixed all spelling errors